src/py/flwr/cli/app.py

```diff
@@ -47,5 +47,28 @@
 
 typer_click_object = get_command(app)
 
+
+def print_version(value: bool) -> None:
```

Comment:
There is no need for a separate print method. It is better to handle it directoy in `@app.callback()`.

Reply:
I tried:
```python
@app.callback()
def version(
    version: bool = typer.Option(
        None,
        "-V",
        "--version",
        is_eager=True,
        help="Prints app version.",
    ),
) -> None:
    """Print version."""
    if version:
        typer.secho(f"Flower version: {package_version}", fg="blue")
        raise typer.Exit()
```
and though `flwr --help` shows:
```
╭─ Options ──────────────────────────────────────────────────────────╮
│ --version             -V        Show the version and exit.         │
```
`flwr -V | --version` does not work.

Another option I have is:
```python
@app.command()
def version() -> None:
    """Print version."""
    typer.secho(f"Flower version: {package_version}", fg="blue")
```
which is very simple and would work only `flwr version`.

Reply:
@alanwilter I guess your first approach is not working because of `is_eager=True`. Also I think `invoke_without_command=True` needs to be set for the app typer. Let me know if that fixes the issue. 
Let's use the first approach as with the second one, `flwr --version` won't work.

Reply:
Eagerness was not the issue, but your `invoke_without_command=True` suggestion did the trick.
However, I can't commit my changes, because I added a test file and `src/py/flwr_tool/check_copyright.py` is throwing an error as it expected the new file to have already been committed! If feels like a "race-condition" here...